opencga-master/src/main/java/org/opencb/opencga/master/monitor/daemons/ExecutionDaemon.java

```diff
@@ -500,9 +500,7 @@ protected void checkPendingJobs(List<String> organizationIds) {
 
         for (String organizationId : organizationIds) {
             try (DBIterator<Job> iterator = jobManager.iteratorInOrganization(organizationId, pendingJobsQuery, queryOptions, token)) {
-                while (iterator.hasNext()) {
-                    pendingJobs.add(iterator.next());
-                }
+                pendingJobs.add(iterator.next());
```

Comment:
Just checking one pending job per organization? Why? This seems an important change

Reply:
Yes, it is and I can't tell you why I did it. Restoring old behaviour